### DISCUSSION TITLE
jsonrpc params sign

### DISCUSSION THREAD

**Discussion by iHibro** [filecoin-project/lotus#11115] [General Q&A]
{"jsonrpc":"2.0","id":1,"method":"Filecoin.MpoolPush","params":[{"Message":{"version":0,"from":"FXXXXXX","to":"FXXXXXX","value":"0","method":16,"nonce":xxxxx,"params":"XXXXXX","gasLimit":7948138,"gasPremium":"2347948138","gasFeeCap":"2347948138"},"CID":"baxxxxxxxxx","Signature":{"Type":2,"Data":"XXXXXX"}}]}

这是一个标准的推送WithdrawBalance jsonrpc，但是我不会编码params，我需要编码以下内容：

{
  "AmountRequested": "161978253220510347524"
}

应该是bhor然后base64，但是怎么写都不行，如果可以能提供php或者nodejs的例子，我可以提供 10 USDT 奖励给您。

**Comment by iHibro** [comment]
已解决 lotus chain encode params t01000 16 '{"AmountRequested": "161978253220510347524"}'

感谢。

**Comment by beck-8** [comment]
看晚了，还有这种问题吗？我想赚钱
